com.unity.postprocessing/PostProcessing/Runtime/PostProcessLayer.cs

```diff
@@ -1334,7 +1362,7 @@ void RenderFinalPass(PostProcessRenderContext context, int releaseTargetAfterUse
                 dithering.Render(context);
 
                 ApplyFlip(context, uberSheet.properties);
-                if (context.stereoActive && context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.SinglePassInstanced)
+                if (context.stereoActive && (context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.SinglePassInstanced || context.stereoRenderingMode == PostProcessRenderContext.StereoRenderingMode.SinglePassMultiview))
```

Comment:
Same here, please revert this line, see discussion ( https://github.com/Unity-Technologies/Graphics/pull/5917#discussion_r723683891)